### DISCUSSION TITLE
Lagrange拓展API

### DISCUSSION THREAD

**Discussion by zyckk4** [nonebot/discussions#516] [想法]
Lagrange在OneBot11基础上有一些拓展API。是否可以新搞一个adapter，把Lagrange的拓展API也带上

**Comment by yanyongyu** [comment]
没必要新搞适配器，只需要加上拓展API的类型注释就可以

**Reply by zyckk4** [reply]
毕竟还是有File, Folder, Node三种拓展消息链。还是希望能有一种方法附加之于ob11，然后nonebot商店能提供

**Reply by yanyongyu** [reply]
message segment可以直接继承使用，ob适配器对消息段内数据不敏感
